src/openforms/registrations/contrib/zgw_apis/models.py

```diff
@@ -65,6 +92,16 @@ def apply_defaults_to(self, options):
         options.setdefault("zaaktype", self.zaaktype)
         options.setdefault("informatieobjecttype", self.informatieobjecttype)
         options.setdefault("organisatie_rsin", self.organisatie_rsin)
+        options.setdefault(
+            "zaak_vertrouwelijkheidaanduiding", self.zaak_vertrouwelijkheidaanduiding
+        )
+        # map variable to a key that conforms to Formio naming standards
+        # otherwise the value cannot be changed by the Formio component
+        options.setdefault(
+            "docVertrouwelijkheidaanduiding",
+            self.doc_vertrouwelijkheidaanduiding,
+        )
```

Comment:
can you elaborate? this is backend code, I don't see how this affects formio :thinking: